chai_lab/chai1.py

```diff
@@ -212,26 +213,55 @@ def raise_if_msa_too_deep(msa_depth: int):
 # Inference logic
 
 
+@typecheck
+@dataclass(frozen=True)
+class StructureCandidates:
+    # We provide candidates generated by a model,
+    #   with confidence predictions and ranking scores.
+    # Predicted structure is a candidate with the highest score.
+
+    # locations of CIF files, one file per caondidate
+    cif_paths: list[Path]
+    # scores for each of candidates + info that was used for scoring.
+    ranking_data: list[SampleRanking]
+    # iff MSA are available, we also save a plot as PDF
```

Comment:
micronit: "MSA are available" could be confused with "if the MSA search returned some sequences" whereas I think it should be "iff MSA search was performed"